pyomo/contrib/solver/common/base.py

```diff
@@ -585,10 +610,12 @@ def _solution_handler(
                     legacy_soln.variable['Rc'] = val
 
         legacy_results.solution.insert(legacy_soln)
-        # Timing info was not originally on the legacy results, but we want
-        # to make it accessible to folks who are utilizing the backwards
-        # compatible version.
-        legacy_results.timing_info = results.timing_info
+        # Timing info was not originally on the legacy results, but we
+        # want to make it accessible to folks who are utilizing the
+        # backwards compatible version.  Note that embetting the
```

Comment:
Typo reported upstream
```suggestion
        # backwards compatible version.  Note that embedding the
```